mojave/items/clothing/head/helmet.dm

```diff
@@ -242,23 +249,22 @@
 	dynamic_hair_suffix = ""
 	dynamic_fhair_suffix = ""
 
-/obj/item/clothing/head/helmet/ms13/eyebot
-	name = "eyebot helmet"
-	desc = "A helmet made from the remains of an eyebot. Solid protection, even if it looks silly."
-	icon_state = "eyebot"
-	inhand_icon_state = "eyebot"
-	armor = list("melee" = 50, "bullet" = 35, "laser" = 5, "energy" = 5, "bomb" = 30, "bio" = 10, "rad" = 10, "fire" = 50, "acid" = 10)
-	flags_inv = HIDEEARS|HIDEFACE|HIDEHAIR
-	dynamic_hair_suffix = ""
-	dynamic_fhair_suffix = ""
-
 /obj/item/clothing/head/helmet/ms13/metal
 	name = "metal helmet"
 	desc = "A robust but cumbersome metal helmet. Bogs you down a bit in exchange for it's good all around protection. Metal is known to be conductive however."
```

Comment:
```suggestion
	desc = "A robust metal helmet offering good all around protection. Metal is known to be very conductive however."
```

Given it no longer has slowdown, no need to have the "cumbersome" and "bogs you down" lines.

Reply:
Truuu!